pydis_site/apps/resources/views/resources.py

```diff
@@ -0,0 +1,12 @@
+from django.core.handlers.wsgi import WSGIRequest
+from django.http import HttpResponse
+from django.shortcuts import render
+from django.views import View
+
+
+class ResourcesView(View):
```

Comment:
Did you consider using a [`TemplateView`](https://docs.djangoproject.com/en/3.1/ref/class-based-views/base/#django.views.generic.base.TemplateView) here? It would do away with all of the boilerplate code:

```py
class ResourcesView(TemplateView):
    template_name = "resources/resources.html"
```

Reply:
Woah, I didn't know about `TemplateView` 😅 . Now this use this.